suite-common/graph/src/graphBalanceEvents.ts

```diff
@@ -119,7 +132,8 @@ export const getAccountMovementEvents = async ({
     endOfTimeFrameDate: Date;
     dispatch: ReturnType<typeof useDispatch>;
 }) => {
-    const { coin, identity, descriptor } = account;
+    const { coin, identity, descriptor, tokensFilter, accountKey } = account;
+    const tokenAddress = tokensFilter?.[0]; // This is only for graph on detail screen where we have always only one token
```

Comment:
There `tokenFilter = []` right now for mainnet so it will show only ETH without tokens, but I will address showing tokens movements in ETH graph in next PR.